### PR TITLE
Streamline home page

### PR DESCRIPTION
**Pull request by davidchesnut**
|        Q        |                    A                    |
| --------------- | --------------------------------------- |
| Bug fix?        | no                               |
| New feature?    | no                               |
| New sample?     | no                            |
| Related issues? | no |

## What's in this Pull Request?

Series of changes to reduce the number of files/folders on the repo home page so the readme is more prominent. Also simplify the sample tables layout in the readme. Plus updates to contribution guidelines to be current.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,58 +1,83 @@
-# Contribution guidance
+# Contributing
```
Comment (AlexJerabek) [inline CONTRIBUTING.md:1]:
```suggestion
# Contribute
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,58 +1,83 @@
...
-Please do not open GitHub issues for general support questions as the GitHub list should be used for feature requests and bug reports. This way we can more easily track actual issues or bugs from the code and keep the general discussion separate from the actual code.
+When you submit changes to this repository, please follow these recommendations.
 
-If you have questions about how to use office.js or the Office developer platform, please post your question on [stackoverflow](https://stackoverflow.com). Tag your question with office-js or outlook-web-addins
+* Always fork the repository to your own account for applying modifications.
+* Do not combine multiple changes in one pull request. Please submit separate PRs for each fix, update, or new sample.
+* If you are submitting a typo or documentation fix, you can combine modifications in a single PR where suitable.
 
-## Typos, issues, bugs and contributions
+## Sample naming and structure guidelines
```
Comment (AlexJerabek) [inline CONTRIBUTING.md:24]:
```suggestion
## Sample guidelines
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,58 +1,83 @@
...
 
-## Question or problem?
+## Typos, issues, bugs and contributions
 
-Please do not open GitHub issues for general support questions as the GitHub list should be used for feature requests and bug reports. This way we can more easily track actual issues or bugs from the code and keep the general discussion separate from the actual code.
+When you submit changes to this repository, please follow these recommendations.
 
-If you have questions about how to use office.js or the Office developer platform, please post your question on [stackoverflow](https://stackoverflow.com). Tag your question with office-js or outlook-web-addins
+* Always fork the repository to your own account for applying modifications.
+* Do not combine multiple changes in one pull request. Please submit separate PRs for each fix, update, or new sample.
```
Comment (AlexJerabek) [inline CONTRIBUTING.md:21]: I think the contraction makes this sound less severe.
```suggestion
* Don't combine multiple changes in one pull request. Please submit separate PRs for each fix, update, or new sample.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,58 +1,83 @@
...
+## Sample naming and structure guidelines
 
-When you submit changes to this PnP repository, please follow these recommendations.
+When you are submitting a new sample, use the following guidelines.
 
-* Always fork repository to your own account for applying modifications
-* Do not combine multiple changes to one pull request, please submit for example any samples and documentation updates using separate PRs
-* If you are submitting multiple samples, please create specific PR for each of them
-* If you are submitting typo or documentation fix, you can combine modifications to single PR where suitable
+**Check for existing samples**
```
Comment (AlexJerabek) [inline CONTRIBUTING.md:28]: Consider using H3s instead of bold text.
```suggestion
### Check for existing samples
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,58 +1,83 @@
...
+If you find a similar sample that already exists in the repo, we would prefer that you extend the existing one, rather than submit a new similar sample.
 
-## Sample naming and structure guidelines
+**Create a README.md file**
```
Comment (AlexJerabek) [inline CONTRIBUTING.md:31]: Consider using H3s instead of bold text.
```suggestion
### Create a README.md file
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,58 +1,83 @@
...
-  * Update the image src element according to correct repository name and folder information. If your sample is for example in samples folder and named as react-todo, src element should be updated as https://telemetry.sharepointpnp.com/pnp-officeaddins/samples/react-todo.
-* If you find a similar kind of sample from the existing samples, we would prefer you extend the existing one, rather than submit a new similar sample.
-  * When you update existing samples, please update the README as well with new information on provided changes and with your author details.
-* When you are submitting a new sample solution, please name the sample solution folder accordingly.
-  * Folder should start by identifying JS library used - like "react-", "angular-", "knockout-"
-  * If you are not using any specific JS library, please use "js-" as the prefix for your sample, or use "ts-" if your sample uses TypeScript.
-  * If your solution is demonstrating multiple technologies, please use functional terms as the name for the solution folder.
-* Do not use period/dot in the folder name of the provided sample.
+**Update tracking image**
```
Comment (AlexJerabek) [inline CONTRIBUTING.md:34]: Consider using H3s instead of bold text.
```suggestion
### Update tracking image
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,58 +1,83 @@
...
+The README template contains specific tracking image as a final entry in the page with img tag by default to https://telemetry.sharepointpnp.com/pnp-officeaddins/samples/name-of-sample. This is transparent image, which is used to track anonymous view counts of individual samples in GitHub.
+
+Update the image src element according to correct repository name and folder information. If your sample is for example in samples folder and named as react-todo, src element should be updated as https://telemetry.sharepointpnp.com/pnp-officeaddins/samples/react-todo.
 
-## The Contribution License Agreement
+**Name the solution**
```
Comment (AlexJerabek) [inline CONTRIBUTING.md:39]: Consider using H3s instead of bold text.
```suggestion
### Name the solution
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,58 +1,83 @@
...
-* When you are submitting a new sample solution, please name the sample solution folder accordingly.
-  * Folder should start by identifying JS library used - like "react-", "angular-", "knockout-"
-  * If you are not using any specific JS library, please use "js-" as the prefix for your sample, or use "ts-" if your sample uses TypeScript.
-  * If your solution is demonstrating multiple technologies, please use functional terms as the name for the solution folder.
-* Do not use period/dot in the folder name of the provided sample.
+**Update tracking image**
+The README template contains specific tracking image as a final entry in the page with img tag by default to https://telemetry.sharepointpnp.com/pnp-officeaddins/samples/name-of-sample. This is transparent image, which is used to track anonymous view counts of individual samples in GitHub.
+
+Update the image src element according to correct repository name and folder information. If your sample is for example in samples folder and named as react-todo, src element should be updated as https://telemetry.sharepointpnp.com/pnp-officeaddins/samples/react-todo.
```
Comment (AlexJerabek) [inline CONTRIBUTING.md:37]: Some formatting suggestions for clarity.
```suggestion
Update the image `src` element according to correct repository name and folder information. For example, if your sample is **/samples** folder and named as "react-todo", `src` element should be updated as https://telemetry.sharepointpnp.com/pnp-officeaddins/samples/react-todo.
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -1,58 +1,83 @@
...
 
-## The Contribution License Agreement
+**Name the solution**
+When you are submitting a new sample solution, please name the sample solution folder accordingly.
 
-For most contributions, you'll be asked to sign a Contribution License Agreement (CLA). This will happen when you submit a pull request. Microsoft will send a link to the CLA to sign via email. Once you sign the CLA, your pull request can proceed. Read the CLA carefully, because you may need to have your employer sign it.
+* The solution folder should be under the **/Samples** folder of this repo.
+* Folder name should be in the format [product]-[scenario]-[platform]. For example, a sample that shows how to create data-bound Excel tables using React, should be named "excel-data-bound-tables-react". The names are all lowercase.
+* If your solution is demonstrating multiple technologies, please use functional terms as the name for the solution folder.
+* Do not use period/dot in the folder name of the provided sample.
```
Comment (AlexJerabek) [inline CONTRIBUTING.md:45]: We should avoid the `/` as "or".
```suggestion
* Do not use a period in the folder name of the provided sample.
```

File: README.md [hunk 10]
```diff
@@ -1,44 +1,52 @@
...
+| [Use storage techniques to access data from an Office Add-in when offline](https://github.com/OfficeDev/PnP-OfficeAddins/tree/main/Samples/Excel.OfflineStorageAddin) | Demonstrates how you can implement localStorage to enable limited functionality for your Office Add-in when a user experiences lost connection. |
+| [Custom function batching pattern](https://github.com/OfficeDev/PnP-OfficeAddins/tree/main/Excel-custom-functions/Batching)| Batch multiple calls into a single call to reduce the number of network calls to a remote service.|
 
-## Auth samples
+## Shared JavaScript runtime
+
+Check out these samples if you want to take advantage of the [shared runtime](https://docs.microsoft.com/office/dev/add-ins/develop/configure-your-add-in-to-use-a-shared-runtime) for your Office Add-in.
 
 | Date               | Name           | Description  |
 | ------------------ | -------------- | ------------ |
```
Comment (AlexJerabek) [inline README.md:39]: Shouldn't we remove the "Date" column here?

File: README.md [hunk 11]
```diff
@@ -1,44 +1,52 @@
-# Office Add-ins Patterns and Practices (PnP)
+# Office Add-ins code samples
```
Comment (AlexJerabek) [inline README.md:1]: How are these sorted? I'm more curious than anything.

**Review by AlexJerabek — APPROVED** [review]
Approving with some feedback.

File: README.md (hunk 11 above)
Comment (davidchesnut) [reply README.md:1]: Kind of by popularity at the moment. Definitely open to ideas on this. :-)

**Pull request merged**